intermediate/indexing/advanced-indexing.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>Conceptually, would it make more sense here if we used <code>lon=target_lon</code> because we'd most likely want that to be on the <code>degrees_east</code> dim?  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Yes and I updated the notebook to reflect this.